sample/CustomHandlerRetry/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "CustomHandlerRetry",
+    "version": "0.0.1",
+    "private": false,
+    "dependencies": {
```

Comment:
Add "express" and "body-parser" to the "dependencies" section here. Then run `npm install` on your local machine - it should generate a "package-lock.json" file which you also want to include in your PR